Dell/recovery_backend.py

```diff
@@ -959,108 +959,74 @@ def create_ubuntu(self, recovery, revision, iso, platform, no_update, sender=Non
         self.xml_obj.replace_node_contents('generator', check_version())
         self.xml_obj.write_xml(os.path.join(tmpdir, 'bto.xml'))
 
-        #Arg list
-        xorrisoargs = ['xorriso',
-                       '-as', 'mkisofs',
-                       '-R',
-                       '-r',
-                       '-J',
-                       '-joliet-long',
-                       '-l',
-                       '-iso-level', '3',
-                       '-sysid', 'Linux',
-                       '-V', 'Dell Recovery Media',
-                       '-A', 'Dell Recovery',
-                       '-p', 'Dell',
-                       '-publisher', 'Dell',
-                       '-o', iso,
-                       '-m', '*.exe',
-                       '-m', '*.sys',
-                       '-m', '*.SDR',
-                       '-m', 'SDR',
-                       '-m', 'syslinux',
-                       '-m', 'syslinux.cfg',
-                       '-m', os.path.join(mntdir, 'bto.xml'),
-                       '-m', os.path.join(mntdir, 'bto_version')]
+        if not os.path.exists('/usr/share/cd-boot-images-amd64/xorriso-cmd.txt'):
+            raise CreateFailed("cd-boot-images-amd64 is not installed")
```

Comment:
How to backward support for old distro that don't use `cd-boot-images-amd64`?

Reply:
Why do we need to backward support for old distros on the latest distro?